src/backend/vulkan/src/lib.rs

```diff
@@ -63,6 +63,21 @@ pub struct Instance {
     pub surface_extensions: Vec<&'static str>,
 }
 
+fn map_queue_type(flags: vk::QueueFlags) -> QueueType {
+    if flags.intersects(vk::QUEUE_GRAPHICS_BIT | vk::QUEUE_COMPUTE_BIT) { // TRANSER_BIT optional
```

Comment:
The bitflag implementation of `ash` doesn't provide `contains` (https://docs.rs/ash/0.18.0/ash/vk/types/struct.QueueFlags.html)